targets/ath79-generic

```diff
@@ -380,6 +380,11 @@ device('siemens-ws-ap3610', 'siemens_ws-ap3610', {
 
 -- Sophos
 
+device('sophos-ap15', 'sophos_ap15', {
+	packages = ATH10K_PACKAGES_QCA9880,
+	factory = false,
+})
```

Comment:
As Serial is not external but  requires disassembly, this device would be marked as broken (add a comment about this circumstance so we can be more granular about this in the future).

Reply:
Well, you don't need serial to flash the device. Serial is only needed if you forgot to setup an SSH key/password because due to the device having no button you wouldn't be able to bring it into config mode again.

The device still can be flashed without opening it via the free of cost Home Edition of the Sophos Firewall, it enables you to SSH into the stock firmware and flash the firmware with mtd from there. Exactly the same as with the other Sophos APs.

Therefore I'm unsure if it really needs the broken flag...

Reply:
@kpanic23 Please add the broken flag, as @herbetom explained the re-entry of the setup-mode is problematic (And will be extended in the future to allow such cases.